Products/ZenModel/migrate/addToolboxLogsToKibana.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import re
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+
+sm.require("1.1.5")
+
+class AddToolboxLogsToKibana(Migrate.Step):
+    "Pull logs provided by toolbox utilities to logstash."
+
+    version = Migrate.Version(115, 0, 0)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+
+        log_paths = dict(
+                        zodbscan="/opt/zenoss/log/toolbox/zodbscan.log",
+                        zennetworkclean="/opt/zenoss/log/toolbox/zennetworkclean.log",
+                        zenindextool="/opt/zenoss/log/toolbox/zenindextool.log",
+                        zencheckdbstats="/opt/zenoss/log/toolbox/zencheckdbstats.log",
+                        zencatalogscan="/opt/zenoss/log/toolbox/zencatalogscan.log",
+                        findposkeyerror="/opt/zenoss/log/toolbox/findposkeyerror.log",
+                        zenrelationscan="/opt/zenoss/log/toolbox/zenrelationscan.log"
+        )
+
+        new_logs = []
+        for name in log_paths.keys():
+            logType = "toolbox_{0}_logs".format(name)
+            new_logs.append(sm.logconfig.LogConfig(
+                                 path=log_paths[name],
+                                 logType=logType,
+                                 filters=['pythondaemon'],
+                                 logTags=None, 
+                                 isAudit=False)
+            )     
+
+        zopes = filter(lambda s: s.name == "Zope", ctx.services)
+        for zope in zopes:
+            zope.logConfigs.extend(new_logs)
```

Comment:
The migration logic needs to be idempotent. This will break in cases where the `zope.logConfigs` already contains these logs.  In other words, the logic needs to be something like - only add these new logs, if zope.logConfigs does not contain any logs from the `/opt/zenoss/log/toolbox` dirctory